src/macros.js

```diff
@@ -298,3 +298,6 @@ defineMacro("\\simcoloncolon", "\\sim\\mathrel{\\mkern-1.2mu}\\dblcolon");
 defineMacro("\\approxcolon", "\\approx\\mathrel{\\mkern-1.2mu}\\vcentcolon");
 defineMacro("\\approxcoloncolon",
             "\\approx\\mathrel{\\mkern-1.2mu}\\dblcolon");
+
+defineMacro("\\limsup", "\\DOTSB\\mathop{\\operatorname{lim\\,sup}}\\limits");
+defineMacro("\\liminf", "\\DOTSB\\mathop{\\operatorname{lim\\,inf}}\\limits");
```

Comment:
[latex.ltx](https://www.tug.org/svn/texlive/trunk/Master/texmf-dist/tex/latex/base/latex.ltx?view=co) seems to indicate that `\mathop` has limits defined by default.  Perhaps that's something we should consider adopting.

Reply:
TODO: remove `\DOTSB`

Reply:
Why remove `\DOTSB`? Maybe I'm missing some context -- if this is in the definition in AMS LaTeX, it makes sense to keep it here, so preceding `\dots` behave correctly.

Reply:
Hmm... maybe I saw the `\DOTSB` when I was inspecting commands using pidflatex as described in https://github.com/Khan/KaTeX/wiki/Examining-TeX.

Reply:
Hmm, I checked the source in `amsopn.sty`, and uses of `\DOTSB` in `amsmath.sty`, and checked in pdflatex... and I now agree that `\DOTSB` should be removed. 😄